spark3-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -73,6 +73,7 @@ statement
     | ALTER TABLE multipartIdentifier WRITE writeSpec                                       #setWriteDistributionAndOrdering
     | ALTER TABLE multipartIdentifier SET IDENTIFIER_KW FIELDS fieldList                    #setIdentifierFields
     | ALTER TABLE multipartIdentifier DROP IDENTIFIER_KW FIELDS fieldList                   #dropIdentifierFields
+    | ALTER TABLE multipartIdentifier ENCRYPTED WITH (KEK | MEK) KEY keyId                  #createTableEncKey
```

Comment:
This is the problem with the `ALTER` syntax.. Currently, we don't have a support for converting an existing regular table to an encrypted table. We will have it in the future, but it will be an explicit and massive data-crunching procedure. For now, we focus on a simpler and more mainstream goal - creating an empty table with an encryption key, so all future data writes end up in encrypted files. It seems to me the `CREATE` syntax is a better fit here..

Regarding dropping the key(s) - I'm not aware of usecases for such functionality. It's possible to think of some, but given the ambiguity and complexity of this function, I'd suggest considering it at a later stage.

Reply:
Thanks for the explanation @ggershinsky. Would suggest to drop change of `ALTER TABLE`.